File: Ruby/1012.rb
def triangulo(a, c)
    return (a * c) / 2
end
def circulo(c)
    pi = 3.14159
    return pi * (c ** 2)
end
def trapezio(a, b, c)
    return ( (a+b) * c ) / 2
end
def quadrado(b)
    return b ** 2
end
def retangulo(a, b)
    return a * b
end

a, b, c = gets.chomp.split
a = a.to_f
b = b.to_f
c = c.to_f

puts ("TRIANGULO: %0.3f" % triangulo(a, c))
puts ("CIRCULO: %0.3f" % circulo(c))
puts ("TRAPEZIO: %0.3f" % trapezio(a, b, c))
puts ("QUADRADO: %0.3f" % quadrado(b))
puts ("RETANGULO: %0.3f" % retangulo(a, b))

File: Python/1012.py
def triangulo(a, c):
    return (a * c) / 2
def circulo(c):
    pi = 3.14159
    return pi * (c ** 2)
def trapezio(a, b, c):
    return ( (a+b) * c ) / 2
def quadrado (b):
    return b ** 2
def retangulo(a, b):
    return a * b

a, b, c = input().split()
a = float(a)
b = float(b)
c = float(c)

print(f"TRIANGULO: {triangulo(a, c):.3f}")
print(f"CIRCULO: {circulo(c):.3f}")
print(f"TRAPEZIO: {trapezio(a, b, c):.3f}")
print(f"QUADRADO: {quadrado(b):.3f}")
print(f"RETANGULO: {retangulo(a, b):.3f}")